docs/open-source-tizen/tools/gbs/gbs-chroot.md

```diff
@@ -1,45 +1,46 @@
 # gbs chroot
 
+Use the `chroot` subcommand to chroot to the build root directory, which is generated by the GBS build.
 
-The subcommand 'chroot' allows users to chroot to the buildroot directory, which is generated by gbs build. You can get the basic usage of gbs chroot using:
+For command usage details, enter:
 
 ```bash
 $ gbs chroot --help
 ```
 
-> **Note**
->  The default location of the build root is located at: ~/GBS-ROOT/local/scratch.{arch}.*, which will be different if the -B option is specified while running gbs build Examples:
+> **Note**  
+>  The default location of the build root is located in `~/GBS-ROOT/local/scratch.{arch}.*`. The location is different if the `-B` option is specified while running the GBS build.
 
-- Create build root with more extra packages to the build root
+Examples of command usage:
```

Comment:
I just copied and pasted the original docs.
If you want to change gbs command manual, would you use analogous or same phrase?

Reply:
We have tried to keep the command topics as similar as possible. Unfortunately, they have very different content, so there are limits to our efforts.

We need some kind of sentence to introduce the examples.
Would it be OK to use "Examples of command usage:" in all the topics that have this kind of example list, or do you just want "Examples:"?